### ISSUE TITLE
OMXPLAYER on 64bit architecture ( aarch64 )

### ISSUE THREAD

**Issue by norterj** [popcornmix/omxplayer#761]
Hello,
I'm using Ubuntu 18.04 64bit on RPi ( 3 and 4 ) and I can't find any way to install omxplayer on it, I tried to compile but it gives me lot of errors.

Any suggestions?

Thank' you.

**Comment by popcornmix** [comment]
I don't believe there are any plans to support openmax (which is a dead api) on aarch64.
We're generally recommending users to migrate from omxplayer to VLC.
VLC has hardware acceleration, supports hevc and software codecs and has many more features than omxplayer.
